### PR TITLE
Feat/met 4419 Parse the right statements and save the category in the DB

### PR DESCRIPTION
**Pull request by JoanaCMS**

### CODE REVIEW

File: statistics-dashboard-common/src/main/java/eu/europeana/statistics/dashboard/common/iternal/RightsCategory.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
+package eu.europeana.statistics.dashboard.common.iternal;
+
```
Comment (jeortizquan) [inline statistics-dashboard-common/src/main/java/eu/europeana/statistics/dashboard/common/iternal/RightsCategory.java:2]: package name iternal? i think you intended ```internal```

Comment (JoanaCMS) [reply statistics-dashboard-common/src/main/java/eu/europeana/statistics/dashboard/common/iternal/RightsCategory.java:2]: Uh oh, this needs to be fixed :p

Comment (jeortizquan) [reply statistics-dashboard-common/src/main/java/eu/europeana/statistics/dashboard/common/iternal/RightsCategory.java:2]: remember to add unit test to this new class.

File: statistics-dashboard-common/src/main/java/eu/europeana/statistics/dashboard/common/iternal/RightsCategory.java [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    CC_BY_NC_SA("CC BY-NC-SA", "^https?[:]//creativecommons[.]org/licen[c,s]es/by-nc-sa/[0-9][.][0-9](/[a-z]{2})?/?$"),
+    CC_BY_NC_ND("CC BY-NC-ND", "^https?[:]//creativecommons[.]org/licen[c,s]es/by-nc-nd/[0-9][.][0-9](/[a-z]{2})?/?$"),
+    NO_COPYRIGHT_NON_COMMERCIAL("No Copyright - Non Commercial Re-Use Only", "^https?[:]//rightsstatements[.]org/vocab/NoC-NC/[0-9][.][0-9](/[a-z]{2})?/?$"),
+    NO_COPYRIGHT_OTHER_KNOWN_LEGAL("No Copyright - Other Known Legal Restriction", "^https?[:]//rightsstatements[.]org/vocab/NoC-NC/[0-9][.][0-9](/[a-z]{2})?/?$"),
+    IN_COPYRIGHT_EDUCATIONAL("In Copyright - Educational Use Permitted", "^https?[:]//rightsstatements[.]org/vocab/InC-EDU/[0-9][.][0-9](/[a-z]{2})?/?$"),
+    IN_COPYRIGHT_EU("In Copyright - EU Orphan Work", "^https?[:]//rightsstatements[.]org/vocab/InC-OW-EU/[0-9][.][0-9](/[a-z]{2})?/?$"),
+    IN_COPYRIGHT("In Copyright", "^https?[:]//rightsstatements[.]org/vocab/InC/[0-9][.][0-9](/[a-z]{2})?/?$"),
+    COPYRIGHT_NOT_EVALUATED("Copyright Not Evaluated", "^https?[:]//rightsstatements[.]org/vocab/CNE/[0-9][.][0-9](/[a-z]{2})?/?$"),
+    UNKNOWN("Unknown", "");
+
```
[the remark this answers is not in the document]
Comment (JoanaCMS) [reply statistics-dashboard-common/src/main/java/eu/europeana/statistics/dashboard/common/iternal/RightsCategory.java:24]: After some discussion with other colleagues, we have to decided to keep it case sensitive

File: statistics-dashboard-common/src/main/java/eu/europeana/statistics/dashboard/common/iternal/RightsCategory.java (hunk 1 above)
Comment (JoanaCMS) [reply statistics-dashboard-common/src/main/java/eu/europeana/statistics/dashboard/common/iternal/RightsCategory.java]: Added

**Review by stzanakis — APPROVED** [review]

**Comment by JoanaCMS** [comment]
This PR was approved and merged

**Pull request closed without merging**